apps/frontend/src/modules/groups/slices/group.slice.ts

```diff
@@ -22,6 +28,8 @@ const initialState: State = {
 	groupCreateStatus: DataStatus.IDLE,
 	groups: [],
 	groupsTotalCount: 0,
+	page: 1,
+	pageSize: 10,
```

Comment:
As we need to handle pagination, it's too complicated to imitate it in reducer. Instead of this let's just do the same trick as when we close modals,  and call onPageSet with the initial page value on success, which will trigger the request and update the group. We want to keep the same pageSize and update page to the initial value.

```suggestion
```

Reply:
I deleted the pagination code from the slice (except for `groupsTotalCount`), and it is working fine without any additional changes. Should I still modify this?